package.json

```diff
@@ -23,6 +23,7 @@
     "build:watch": "yarn build:ts -w",
     "clean:generated": "find ./packages -name \"*.d.ts\" -type f -delete && find ./packages -name \"*.tsbuildinfo\" -type f -delete",
     "deploy-storybook": "storybook-to-ghpages",
+    "generate:component": "hygen component new",
```

Comment:
The problem with this is that we cannot use commands like `hygen component new:stories` (this will only add a stories file)...

Reply:
Deleted